### DISCUSSION TITLE
Persist scroll not working with inline style

### DISCUSSION THREAD

**Discussion by pjadanowski** [livewire/livewire#8368] [7. Bugs (archived)]
### Livewire version

v3.4.11

### Laravel version

v11.5

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

1. add `@persist('scrollbar')` to wrap the parent div
2. add `wire:scroll` to parent div
3. add some inline styles to like ` style="height: calc(100vh - 115px)"`
4. scroll down in the list of elements -> the error comes immediately 
```
livewire.js?id=44144c23:1 Alpine Expression Error: Unexpected token '}'

Expression: "$wire."
```

after taking out `styles` there is no more error, but the scroll position is also not persisted
![image](https://github.com/livewire/livewire/assets/16418436/a7daef0c-8af7-43fd-8ca8-21a3df15fd38)

### Repository or Wirebox URL

https://wirebox.app/b/x5lrj

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by francoism90** [comment]
Seems we both have the same issue: https://github.com/livewire/livewire/discussions/8396

It does work fine for me however.
